packages/adapters-library/src/defiProvider.ts

```diff
@@ -77,12 +78,15 @@ export class DefiProvider {
   private adaptersControllerWithoutPrices: AdaptersController
 
   private metadataProviders: Record<Chain, IMetadataProvider>
+  private unwrapCache: IUnwrapCache
 
   constructor(
     config?: DeepPartial<IConfig>,
     metadataProviders?: Record<Chain, IMetadataProvider>,
+    unwrapCacheProvider?: IUnwrapCacheProvider,
   ) {
     this.metadataProviders = metadataProviders ?? buildMetadataProviders()
+    this.unwrapCache = new UnwrapCache(unwrapCacheProvider)
```

Comment:
No provider by default.

Passing an in-memory cache will result in some snapshots changing due to block differences being in the 10 minute range.